tracer/src/Datadog.Trace/Logging/TracerFlare/TracerFlareRequestFactory.cs

```diff
@@ -0,0 +1,101 @@
+﻿// <copyright file="TracerFlareRequestFactory.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+#nullable enable
+
+using System;
+using System.IO;
+using System.Threading.Tasks;
+using Datadog.Trace.Util;
+
+namespace Datadog.Trace.Logging.TracerFlare;
+
+internal class TracerFlareRequestFactory
+{
+    internal const string Boundary = "--83CAD6AA-8A24-462C-8B3D-FF9CC683B51B";
```

Comment:
What exactly is this `Boundary` value?

Reply:
This is the value that is used between different "parts" of a multipart form: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Type#boundary